SH4RE/SH4RE/Search/FilterSheetView.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  FilterSheetView.swift
+//  SH4RE
+//
+//  Created by Americo on 2023-01-19.
+//
+
+import SwiftUI
+import Combine
+
+struct LocationEntryField: View {
+    @Binding var location: String
+    var body: some View {
+        HStack {
+            TextField("Location", text: $location)
+                .textFieldStyle(
+                    locationInputStyle(
+                        button: Button(action:{
+                            // Currently no action, have to set up location
+                            // This could be a staring point https://www.youtube.com/watch?v=cOD1l2lv2Jw&ab_channel=azamsharp
+                            // Additionally, check out the file "CurrentLocationButton"
+                        }, label:{
+                            Image(systemName: "scope")
+                        })
+                    )
+                )
+        }
+    }
+}
+
+struct FilterSheetView: View {
+    
+    var dropDownList = ["Film & Photography", "Audio Visual Equipment", "Projectors & Screens", "Drones", "DJ Equipment", "Transport", "Storage", "Electronics", "Party & Events", "Sports", "Musical Instruments", "Home, Office & Garden", "Holiday & Travel", "Clothing"]
+    @State private var dropDownSelection: String = ""
+
+    @State private var location: String = ""
+    @State private var minPrice: String = ""
+    @State private var maxPrice: String = ""
```

Comment:
can these be ints if only ints r being entered? im not sure but jw, might be better for type checking

Reply:
unfortunately not. you would think but they have to be strings since text field entry can technically be anything. some stupid reason, apple has not mad e a"number entry only" field. it is actually quite difficult to filter only by numbers, and basically requires you watching for input, then deleting anything that isn't a number from the entry before updating the view, loser code alert ⚠️